.github/workflows/java-8-builds.yml

```diff
@@ -0,0 +1,30 @@
+name: Java 8 CI (MC 1.13-1.16)
```

Comment:
While we could put these in the same workflow (skriptTestFull), I think it's not a bad idea to have them in separate workflows for clarity 👍